server/src/test/java/io/crate/integrationtests/UserDefinedFunctionsIntegrationTest.java

```diff
@@ -299,6 +300,32 @@ public void test_pg_function_is_visible() throws Exception {
         assertThat(response.rows()[0][0], is(false));
     }
 
+    @Test
+    public void test_pg_get_function_result() throws Exception {
+        TypeSignature returnTypeSig = TypeSignature.parseTypeSignature("array(array(integer))");
+        String returnType = returnTypeSig.toString();
+        Signature signature = Signature
+            .builder()
+            .kind(FunctionType.SCALAR)
+            .name(new FunctionName(Schemas.DOC_SCHEMA_NAME, "make_2d_array"))
+            .argumentTypes(TypeSignature.parseTypeSignature("integer"))
```

Comment:
will be a better choise if there would be some kind of refactoring
```suggestion
            .argumentTypes(DataTypes.INTEGER.getTypeSignature()"))
```